main.go

```diff
@@ -122,6 +123,19 @@ func main() {
 		requireLogin.RequireLogin(home.HomeHandler),
 	)
 
+	// Export db page records to Alglolia index
+	log.Println("Exporting records to Algolia servers...")
+	pages, err := databaseActions.GetPages()
```

Comment:
I agree. Originally I had algoliaUtil retrieve the pages but there were some perceived dependency issues (algoliaUtil was being imported into databaseActions so that when a page is updated in databaseActions then a request would be made to update the Algolia index). I believe I moved that logic out of databaseActions so it should be fine.